### PR TITLE
Rename to Illumina Annotator

### PR DESCRIPTION
**Pull request by Fahd-Siddiqui**

### CODE REVIEW

**Comment by Fahd-Siddiqui** [comment]
image

**Comment by rajatshuvro** [comment]
The Logo should not go past the Left panel. And it also looks too large on the main panel.

**Comment by Fahd-Siddiqui** [comment]
> The Logo should not go past the Left panel. And it also looks too large on the main panel.

Agree, the logo you provided was looking really weird. 

On your second point, agreed. I am going to fix it.

File: docs/introduction/TestIlluminaAnnotator.sh [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-NIRVANA_BUILD_ZIP=$1
-NIRVANA_ROOT=~/NirvanaTest
-NIRVANA_BUILD_DIR=$NIRVANA_ROOT/build
-NIRVANA_BIN=$NIRVANA_BUILD_DIR/Nirvana.dll
-DOWNLOADER_BIN=$NIRVANA_BUILD_DIR/Downloader.dll
-DATA_DIR=$NIRVANA_ROOT/Data
+ILLUMINA_ANNOTATOR_BUILD_ZIP=$1
+ILLUMINA_ANNOTATOR_ROOT=~/IlluminaAnnotatorTest
+ILLUMINA_ANNOTATOR_BUILD_DIR=$ILLUMINA_ANNOTATOR_ROOT/build
+ILLUMINA_ANNOTATOR_BIN=$ILLUMINA_ANNOTATOR_BUILD_DIR/IlluminaAnnotator.dll
```
Comment (rajatshuvro) [inline docs/introduction/TestIlluminaAnnotator.sh]: Let's use Annotator.dll

File: docs/introduction/covid19.md [hunk 2]
```diff
@@ -11,39 +11,39 @@ In addition to normal transcript annotation, we also supply:
...
 ```
 
-## Running Nirvana
+## Running Illumina Annotator
 
 Once you have downloaded the data sets, use the following command to annotate your VCF:
 
 ```bash
-dotnet bin/Release/netcoreapp2.1/Nirvana.dll \
+dotnet IlluminaAnnotator.dll \
```
Comment (rajatshuvro) [inline docs/introduction/covid19.md]: Let's use Annotator.dll

File: docs/introduction/getting-started.md [hunk 3]
```diff
@@ -86,18 +82,18 @@ From time to time, you can re-run the Downloader to get the latest annotation fi
...
 ```
 
-## Running Nirvana
+## Running Illumina Annotator
 
 Once you have downloaded the data sets, use the following command to annotate your VCF:
 
 ```bash
-dotnet bin/Release/net6.0/Nirvana.dll \
+dotnet IlluminaAnnotator.dll \
```
Comment (rajatshuvro) [inline docs/introduction/getting-started.md]: Let's use Annotator.dll

File: docs/introduction/getting-started.md [hunk 4]
```diff
@@ -137,18 +133,18 @@ Annotation                                          00:00:00.8       18.5 %
...
-dotnet bin/Release/net6.0/Nirvana.dll
+dotnet IlluminaAnnotator.dll
 ---------------------------------------------------------------------------
-Nirvana                                             (c) 2023 Illumina, Inc.
+IlluminaAnnotator                                   (c) 2020 Illumina, Inc.
 Stromberg, Roy, Platzer, Siddiqui, Ouyang, et al                     3.21.0
 ---------------------------------------------------------------------------
 
-USAGE: dotnet Nirvana.dll -i <vcf path> -c <cache dir> --sd <sa dir> -r <ref path> -o <base output filename>
+USAGE: dotnet IlluminaAnnotator.dll -i <vcf path> -c <cache dir> --sd <sa dir> -r <ref path> -o <base output filename>
```
Comment (rajatshuvro) [inline docs/introduction/getting-started.md]: Let's use Annotator.dll

File: docs/introduction/getting-started.md [hunk 5]
```diff
@@ -177,17 +173,17 @@ OPTIONS:
 Supplementary annotation version: 69, Reference version: 7
 ```
 ### Specifying annotation sources
-By default, Nirvana will use all available data sources. However, the user can customize the set of sources using the `--sources|-s` option. If an unknown source is specified, a warning message will be printed.
+By default, Illumina Annotator will use all available data sources. However, the user can customize the set of sources using the `--sources|-s` option. If an unknown source is specified, a warning message will be printed.
 ```bash
-dotnet bin/Release/net6.0/Nirvana.dll \
+dotnet IlluminaAnnotator.dll \
```
Comment (rajatshuvro) [inline docs/introduction/getting-started.md]: Let's use Annotator.dll

**Review by rajatshuvro — CHANGES_REQUESTED** [review]
Let's use Annotator.dll

**Comment by rajatshuvro** [comment]
I see the local url says "NirvanaDocumentation"

**Review by rajatshuvro — APPROVED** [review]

**Pull request merged**